README.md

```diff
@@ -14,7 +14,7 @@ Additional description about the project and its features.
 
 ## Live Demo
 
-[Live Demo Link](https://raw.githack.com/SaidRasinlic/MicroversePortfolio/main/index.html)
+[Live Demo Link](https://raw.githack.com/SaidRasinlic/MicroversePortfolio/desktop-version/index.html)
 
 
 ## Getting Started
```

Comment:
- [ ] **Required** This PR should be opened so I can have access to the required tools to perform a code review.